test/functional/interface_rest.py

```
@@ -27,6 +27,7 @@
     MiniWallet,
     getnewdestination,
 )
+from typing import Optional, Union, Dict, Any
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)